### ISSUE TITLE
Recompiling Keras Model in Callback during Training

### ISSUE THREAD

**Issue by AndreasPangerl** [keras-team/keras#13368]
Hi,

I'm trying to freeze some layers and update the loss weights in a Callback during training. But I realized that recompiling the model isn't working in the callback. Is there some way, to recompile it during training?

Thanks and Best Regards,
Andreas
